### ISSUE TITLE
RC1 Spring Boot starter fails at mvn clean install

### ISSUE THREAD

**Issue by alump** [vaadin/skeleton-starter-flow-spring#177]
Happens with current Intellij, using it's integrated Maven with Java 12. Changing maven to external fixes the issue. So low priority.

**Steps to reproduce:**
1. Take RC1 version of spring boot starter (I used git as not available at site)
2. Run `mvn clean install` on project

**Expected outcome:**
It builds project

**Actual outcome:**
Fails to missing logger implementation dependency? See attached output.

**Comment by alump** [comment]
[mvn-clean-install.log](https://github.com/vaadin/skeleton-starter-flow-spring/files/3307791/mvn-clean-install.log)

**Comment by alump** [comment]
I can run clean install with java 8, but not with java 12.

**Comment by caalador** [comment]
There hasn't been a rc1 spring boot starter yet for rc1 as the changes went in around 8am thursday Finnish time.

The exception seems like there might be some conflict in the maven plugins.  
Clean and update should fix this for instance `mvn clean verify -U`  

Else also removing the ´node_modules` folder and package_lock.json file.
Also the webpack.config.js needs removing to update it for the 2.0.0 final version after which it shouldn't need regeneration.

**Comment by denis-anisimov** [comment]
Could you please verify that Mikael's suggestion works : `mvn clean verify -U` ?

**Comment by alump** [comment]
Fails same way. See log.

[with-u.log](https://github.com/vaadin/skeleton-starter-flow-spring/files/3311440/with-u.log)

**Comment by alump** [comment]
So I ran `clean verify -U`, got this output...
[clean-verify-with-u.log](https://github.com/vaadin/skeleton-starter-flow-spring/files/3311462/clean-verify-with-u.log)

Then I tried to run `clean install` after it and it fails same way...
[clean-install-after-clean-verify-with-u.log](https://github.com/vaadin/skeleton-starter-flow-spring/files/3311470/clean-install-after-clean-verify-with-u.log)

**Comment by denis-anisimov** [comment]
OK, looks like Java 12 related bug.

**Comment by caalador** [comment]
I can't reproduce this issue with Maven 3.6.0 and JDK 12.0.1

What maven version are you running with? Have you tried `mvn dependency:purge-local-repository` or clening the whoe .m2 repository

**Comment by alump** [comment]
It happens with bundled Maven IDEA provides, my IDEA shows it as 3.3.9.

I do not have the issue, if i use external Maven (3.6.1)

**Comment by caalador** [comment]
Seems like using ideas 3.3.9 does indeed have an incompatibility within the plugins when doing install. Changing idea to use 3.6.0 makes it pass.
Flow is building the project using 3.5.3 I'll make a test snapshot using 3.3.9 for the install step.

**Comment by caalador** [comment]
Using 3.3.9 to build flow and the plugin didn't affect the end result. 
Only option is to change the maven version that idea uses to another version.

According to the IDEA 2019 progressrepot the bundled maven version was updated to Maven 3.6.0. in EAP2, but it was downgraded back to 3.3.9 in Beta3 with the note:
```
The bundled Maven version has downgraded from 3.6.0 to 3.3.9. The bundled version will be upgraded to 3.6.0 again with the first bug-fix update of IntelliJ IDEA 2019.1.
```

**Comment by mstahv** [comment]
Did a local build with these changes, changed to flow-maven-plugin (as I don't know how to build the vaadin branded version) and it seems to fix the issue:

```
g-10-64-12-109:flow }> git diff
diff --git a/flow-maven-plugin/pom.xml b/flow-maven-plugin/pom.xml
index 106eabed30..a54db69c8f 100644
--- a/flow-maven-plugin/pom.xml
+++ b/flow-maven-plugin/pom.xml
@@ -11,7 +11,7 @@
     <name>Flow Maven plugin</name>
 
     <properties>
-        <maven.version>3.5.4</maven.version>
+        <maven.version>3.3.9</maven.version>
     </properties>
 
     <dependencies>
@@ -42,7 +42,7 @@
         <dependency>
             <groupId>com.github.eirslett</groupId>
             <artifactId>frontend-plugin-core</artifactId>
-            <version>1.6</version>
+            <version>1.7.6</version>
         </dependency>
         <dependency>
             <groupId>org.reflections</groupId>
```

**Comment by mstahv** [comment]
This PR should fix this: https://github.com/vaadin/flow/pull/5978

**Comment by pleku** [comment]
Will be fixed in 14.0.0.rc3
